tensorflow/core/kernels/ragged_range_op.cc

```diff
@@ -91,12 +91,13 @@ class RaggedRangeOp : public OpKernel {
                             Eigen::numext::abs(delta));
       } else {
         // The following is copied from tensorflow::RangeOp::Compute().
+        using U = std::make_unsigned_t<T>;
```

Comment:
Wait, I'm confused here.  This is for non-integer types?  In that case, making a type unsigned doesn't make sense.  I also don't see how it would fix the reported issue.  Can you add a test?